website/server/controllers/api-v3/user.js

```diff
@@ -223,6 +223,10 @@ api.deleteUser = {
 
     await user.remove();
 
+    if (req.body.feedback) {
+      // Send feedback via email
```

Comment:
The code used to send the email can be this:
```
      txnEmail(TECH_ASSISTANCE_EMAIL, 'admin-feedback', [
        {name: 'PROFILE_NAME', content: user.profile.name},
        {name: 'UUID', content: user._id},
        {name: 'EMAIL', content: getUserInfo(user, ['email']).email},
        {name: 'FEEDBACK_SOURCE', content: 'from deletion form')},
        {name: 'FEEDBACK', content: req.body.feedback},
      ]);
```
That's based on similar code in `website/server/libs/payments.js`  (search in that file for `admin-user-subscription-details`) so you can copy related code from that file, e.g., for defining `txnEmail` and `TECH_ASSISTANCE_EMAIL`.

I've created a template in Mandrill called `admin-feedback`, which expects the parameters listed in that code block.

The FEEDBACK_SOURCE parameter should have just the text I've shown in that code block, although if you want to configure it as a constant, that's fine. The string will only ever be used in that one place though. The idea of the FEEDBACK_SOURCE parameter is that in future we can use the same email template in other locations, and adjust only the FEEDBACK_SOURCE value so that the admins who receive the email know what's triggered the feedback.